samples/bluetooth/shell_bt_nus/README.rst

```diff
@@ -17,31 +17,42 @@ Requirements
 
 * One of the following development boards:
 
-  * |nRF5340DK|
-  * |nRF52840DK|
-  * |nRF52DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set1_start
+   :end-before: set1_end
 
 * A second nRF52 Development Kit board (PCA10040) for connecting with bt_nus_shell.py.
-  Alternatively, you can use :ref:`ble_console_readme` for connecting (Linux only).
+  Alternatively, you can use :ref:`ble_console_readme` for connecting, using Linux only.
```

Comment:
Does this "alternatively" mean I don't need the nRF52 board anymore?

Reply:
yes

Reply:
fixed